homeex/src/io/codelex/arithmetic/practice/SumAverageRunningInt.java

```diff
@@ -0,0 +1,16 @@
+package io.codelex.arithmetic.practice;
+
+public class SumAverageRunningInt {
+    public static void main(String[] args) {
+        int sum = 0;
+        int lowerBound = 1;
+        int upperBound = 100;
+
+        for (int number = lowerBound; number <= upperBound; ++number) {
+            sum += number;
+        }
+        double average = (double) sum / upperBound;
```

Comment:
Šis strādās tikai tad, ja lowerBound = 1, ja tas mainās, tad upperBound neatbilst skaitļu skaitam.